geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseConnectionPool.scala

```diff
@@ -76,10 +85,14 @@ object HBaseConnectionPool extends LazyLogging {
   def getConnection(params: java.util.Map[String, Serializable], validate: Boolean): Connection = {
     if (ConnectionParam.exists(params)) {
       ConnectionParam.lookup(params)
+    } else if (params.containsKey("hadoop.configuration")) {
+      val conf = params.get("hadoop.configuration").asInstanceOf[Configuration]
```

Comment:
let's make this an actual data store parameter - but don't add it to the 'getParamterInfo' method, as we don't need it in geoserver

Reply:
hmm, i guess hbase isn't in scope where it's being used in geomesa-jobs... and hadoop isn't in scope in geomesa-index-api

Reply:
i guess just make the key a constant in GeoMesaDataStoreFactory